servers/partner-ci/10.0/zabbix-nightly.yaml

```diff
@@ -0,0 +1,62 @@
+- runner-condition
+- multijob:
+    name: 'Smoke/BVT test for dvs{fuel_release}.{plugin_name}.{plugin_version}'
```

Comment:
this is very strange
zabbix cases are already presented in another PR
need to discuss that